ckanext/who_afro/assets/css/who-afro.css

```
@@ -1008,3 +1425,45 @@ a.tag:hover  {
     margin-top: 20px;
     padding: 10px 0px;
 }
+
+.description {
```

It's quite dangerous creating /using such a generic class name.  You have accidentally styled the text in other parts of the site too. You need to make this specific to the home page i.e...
```suggestion
.homepage .promoted-left .description {
```